### DISCUSSION TITLE
Only 60FPS with 120Hz panel?

### DISCUSSION THREAD

**Discussion by antgustech** [bromite/bromite#1210] [Q&A]
Oneplus 8T has a 120Hz panel. I have enabled FPS counter in the corner in the developer settings. When opening Google Chrome it shows 120FPS. When using Bromite it is limited to 60FPS.

Is there are flag or something I can use to enable it?

**Comment by chromer030** [comment]
Even on unmodified Chromium browser (https://github.com/bromite/chromium/releases) we have this limitation, Google has been added this feature to google-chrome as propriety.

**Reply by antgustech** [reply]
Oh..that is a real shame.

**Reply by chromer030** [reply]
You can use AutoHZ app to force it to 120 Hz.
(https://play.google.com/store/apps/details?id=com.arter97.auto90)

**Reply by antgustech** [reply]
Awesome, it works! Thanks!

**Reply by csagan5** [reply]
> Google has been added this feature to google-chrome as propriety.

source?
